docs/getting-started/clients/web-vault/index.mdx

```diff
@@ -11,10 +11,24 @@ import CodeBlock from "@theme/CodeBlock";
 ## Requirements
 
 - Before you start, you must complete the [Clients repository setup instructions](../index.md).
-
 - If you want to develop against a local server instance, see the
   [Server Setup Guide](../../server/guide.md).
 
+### SSL Certificate
+
+It's recommended to generate a valid SSL certificate for local development rather than using the
+shared development cert. This will prevent browser warnings about invalid certificates and properly
```

Comment:
The shared cert is bitwarden specific, right? we should make this section agnostic or have two versions where only bitwarden one references this.

Reply:
No it's checked in to the repository and is used by anyone.